cla_public/templates/checker/result/_find-legal-adviser.html

```diff
@@ -86,38 +86,38 @@ <h3 class="org-title">
                   </div>
                 </header>
                 <div class="org-details">
-                  {% if item.categories|length %}
-                    <div class="org-categories" role="list">
-                      <div class="sr-only">{{ _('Categories of law') }}</div>
-                      {% for cat in item.categories if cat %}
-                        <span class="org-category" role="listitem">{{ cat }}</span>
-                      {% endfor %}
-                    </div>
-                  {% endif %}
-                  <p>
+                  <p class="govuk-body-s">
                     <span class="sr-only">{{ _('Address') }}:</span>
                     <span class="adr">
                       <span class="street-address">{{ item.location.address }}</span>
                       <span class="city">{{ item.location.city }}</span>
                       <span class="postal-code">{{ item.location.postcode }}</span>
                     </span>
                   </p>
-                  <p>
+                  <p class="govuk-body-s">
                     <span>{{ _('Helpline') }}:</span>
                     <span class="tel">{{ item.telephone }}</span>
                   </p>
                   {% if item.organisation.website %}
-                    <p>
+                    <p class="govuk-body-s">
                       <span>{{ _('Website') }}:</span>
                       {{ Element.link_same_window(item.organisation.website|human_to_url, item.organisation.website|url_to_human, is_external=True, **{'class': 'url'}) }}
                     </p>
                   {% endif %}
+                  {% if item.categories|length %}
+                    <h4 class="govuk-heading-s govuk-!-font-size-16 govuk-!-margin-bottom-2">{{ _('Categories of law') }}</h4>
+                    <ul class="govuk-list govuk-list--bullet govuk-!-font-size-14" role="list">
+                      {% for cat in item.categories if cat %}
+                        <li class="govuk-!-margin-0" role="listitem">{{ cat }}</li>
+                      {% endfor %}
+                    </ul>
+                  {% endif %}
                 </div>
               </li>
             {% endfor %}
           </ul>
         </div>
-        <nav class="search-results-pagination">
+        <nav class="search-results-pagination" style="display:none; /*reason:this-is-dormant*/">
```

Comment:
What does dormant mean here? Does it get populated and shown by ajax? Or is it currently never populated or visible in the current codebase?

Reply:
I haven't investigated thoroughly.  Pagination isn't visible, but the container is (and is always empty).  It was eating into the list space, so I added `display:none` to hide it.  I was loathed to delete it in case we need pagination added later - so I added that comment.

Reply:
If it's of no use now I'd rather delete it and trust that either it's easy enough to recreate or the devs find its removal and reinstate it from that. I don't like having non-functioning code lying around.